exporters/test/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+if(WITH_CURL)
+  add_executable("curl_http_test" "curl_http_test.cc")
```

Comment:
nit: double quotes are unnecessary here.

Reply:
Thanks, have instead added variable to store the filename.